drivers/sps30/sps30.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (C) 2020 Michel Rottleuthner
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sps30
+ * @brief       Device driver for the Sensirion SPS30 particulate matter sensor
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ * @file
+ */
+#define LOG_LEVEL LOG_DEBUG
+#include "log.h"
+
+#include <errno.h>
+#include <string.h>
+
+#include "checksum/crc8.h"
+#include "sps30.h"
+#include "xtimer.h"
+#include "byteorder.h"
+#include "kernel_defines.h"
+
+/**
+ * @name    SPS30 literal definitions
+ * @{
+ */
+#define SPS30_I2C_ADDR       (0x69) /**< Fixed I2C address for the Sensor */
+#define SPS30_CRC_POLY       (0x31) /**< Polynomial for the CRC calulation */
+#define SPS30_CRC_INIT       (0xFF) /**< Init value for the CRC calulation */
+#define SPS30_MESAURE_MODE   (0x03) /**< Fixed mode byte for start cmd */
+#define SPS30_DUMMY_BYTE     (0x00) /**< Fixed dummy byte for unused bytes */
+#define SPS30_PTR_LEN           (2) /**< Pointer address length in bytes */
+/** @} */
+
+/**
+ * @name    SPS30 I2C commands
```

Comment:
`@name` should only be used with doxygen groups, also `@name` and `@brief` together seems wrong

Reply:
done